src/Database/Concerns/PendingScope.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Database\Concerns;
+
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Scope;
+
+class PendingScope implements Scope
+{
+    /**
+     * All of the extensions to be added to the builder.
+     *
+     * @var string[]
+     */
+    protected $extensions = ['WithPending', 'WithoutPending', 'OnlyPending'];
```

Comment:
@stein-j Is this a common pattern used by Laravel or just something you're using here?

Reply:
I was curious about this before. I searched for the property, and Laravel uses the same thing in [SoftDeletingScope](https://github.com/laravel/framework/blob/9.x/src/Illuminate/Database/Eloquent/SoftDeletingScope.php) (and it looks like Laravel doesn't provide more scopes out of the box). So I think the `$extensions` property is fine here